en/collect/add-entry-using-an-id.md

```diff
@@ -60,6 +60,12 @@ The [International Association for Cryptologic Research](https://www.iacr.org/)
 
 ID search is carried out using the Cryptology ePrint ID.
 
+### JSTOR
+
+Jstor is a digital library of academic content in many disciplines.
+ID search is carried out using either the [stable url](https://support.jstor.org/hc/en-us/articles/115004872507-Citation-Management-Permanently-Linking-to-Content-on-JSTOR) or a DOI.
```

Comment:
Please, clarify what is the "stable url" compared to DOI: [Jstor stable url].

Reply:
The stable url is basically the same as a doi.

Some stable urls only contain the doi(http://www.jstor.org/stable/10.1086/501484 for example)
while most use a common prefix(https://www.jstor.org/stable/90002164  = **10.2307**/90002164)

Reply:
In this help section, that would be too much information. I suggest : "ID search is carried out using either the [Jstor stable url](https://support.jstor.org/hc/en-us/articles/115004872507-Citation-Management-Permanently-Linking-to-Content-on-JSTOR) or the DOI."

Reply:
Which is what if have written.

Reply:
Almost :wink: : I recommended [stable **jstor** url]. Plus "the" or "a" for both.

Reply:
oh, you are correct, fixed.